### PR TITLE
Fix no such module 'DOT' by  pinning GraphViz 0.2.0

### PR DESCRIPTION
**Pull request by yanamura**
- Reverts yonaskolb/XcodeGen#1065
  - issue
    - https://github.com/yonaskolb/XcodeGen/pull/1066#issuecomment-830784929

## Solution
- pinned GraphViz 0.2.0

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -19,7 +19,7 @@ let package = Package(
         .package(url: "https://github.com/tuist/XcodeProj.git", from: "7.18.0"),
         .package(url: "https://github.com/jakeheis/SwiftCLI.git", from: "6.0.0"),
         .package(url: "https://github.com/mxcl/Version", from: "2.0.0"),
-        .package(url: "https://github.com/SwiftDocOrg/GraphViz.git", from: "0.4.0"),
+        .package(url: "https://github.com/SwiftDocOrg/GraphViz.git", from: "0.1.0"),
```
Comment (yonaskolb) [inline Package.swift:22]: To fix the original issue in https://github.com/yonaskolb/XcodeGen/pull/1065, should this be pinned to 0.2?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,9 +2,6 @@
 
 ## Next Version
 
-#### Fixed
-- Fixed no such mocule `DOT` error [#1065](https://github.com/yonaskolb/XcodeGen/pull/1065) @yanamura
```
Comment (yonaskolb) [inline CHANGELOG.md:6]: If we pin the version below I believe this should still be fixed, so you an add it back with this PR, but fix `mocule` -> `module`

File: Package.swift (hunk 1 above)
Comment (yanamura) [reply Package.swift:22]: pinned version in Package.resolved is 0.2.0 , so both 0.1.0 and 0.2.0 are available.

Comment (yonaskolb) [reply Package.swift:22]: Yes, but I think `Package.resolved` isn't used if the package is used as a dependency in another package, which is when the issue occurs

Comment (yanamura) [reply Package.swift:22]: oh, I got it! thanks for your suggestion. I'll fix now

**Review by yonaskolb — APPROVED** [review]

**Pull request merged**